reference/tools/tiup/cluster.md

```diff
@@ -0,0 +1,541 @@
+---
+title: 使用 TiUP 部署运维 TiDB 线上集群
+category: tools
+---
+
+# 使用 TiUP 部署运维 TiDB 线上集群
+
+本文重在介绍如何使用 TiUP 的 cluster 组件，如果需要线上部署的完整步骤，可参考[使用 TiUP 部署 TiDB 集群](/how-to/deploy/orchestrated/tiup.md)。
+
+与 playground 组件用于部署本地集群类似，cluster 组件用于快速部署生产集群。对比 playground，cluster 组件提供了更强大的集群管理功能，包括对集群的升级、缩容、扩容甚至操作、审计等。
+
+cluster 组件的帮助文档如下：
+
+```bash
+tiup cluster
+```
+
+```
+The component `cluster` is not installed; downloading from repository.
+download https://tiup-mirrors.pingcap.com/cluster-v0.4.9-darwin-amd64.tar.gz 15.32 MiB / 15.34 MiB 99.90% 10.04 MiB p/s                                                   
+Starting component `cluster`: /Users/joshua/.tiup/components/cluster/v0.4.9/cluster 
+Deploy a TiDB cluster for production
+
+Usage:
+  tiup cluster [flags]
+  tiup [command]
+
+Available Commands:
+  deploy      部署集群
+  start       启动已部署的集群
+  stop        停止集群
+  restart     重启集群
+  scale-in    集群缩容
+  scale-out   集群扩容
+  destroy     销毁集群
+  upgrade     升级集群
+  exec        在集群的一个或多个机器上执行命令
+  display     获取集群信息
+  list        获取集群列表
+  audit       查看集群操作日志
+  import      导入一个由 TiDB-Ansible 部署的集群
+  edit-config 编辑 TiDB 集群的配置
+  reload      用于必要时重载集群配置
+  patch       使用临时的组件包替换其群上已部署的组件
+  help        打印 Help 信息
+
+Flags:
+  -h, --help              帮助信息
+      --ssh-timeout int   SSH 连接超时时间
+  -y, --yes               跳过所有的确认步骤
+```
+
+## 部署集群
+
+部署集群的命令为 `tiup cluster deploy`，一般用法为：
+
+```bash
+tiup cluster deploy <cluster-name> <version> <topology.yaml> [flags]
+```
+
+该命令需要提供集群的名字、集群使用的 TiDB 版本，以及一个集群的拓扑文件。
+
+拓扑文件的编写可参考[示例](https://github.com/pingcap-incubator/tiup-cluster/blob/master/examples/topology.example.yaml)。以一个最简单的拓扑为例，将下列文件保存为 `/tmp/topology.yaml`：
+
+```yaml
+---
+  
+pd_servers:
+  - host: 172.16.5.134
+    name: pd-134
+  - host: 172.16.5.139
+    name: pd-139
+  - host: 172.16.5.140
+    name: pd-140
+
+tidb_servers:
+  - host: 172.16.5.134
+  - host: 172.16.5.139
+  - host: 172.16.5.140
+
+tikv_servers:
+  - host: 172.16.5.134
+  - host: 172.16.5.139
+  - host: 172.16.5.140
+
+grafana_servers:
+  - host: 172.16.5.134
+
+monitoring_servers:
+  - host: 172.16.5.134
+```
+
+假如我们想要使用 TiDB 的 v4.0.0-rc 版本，集群名字为 `prod-cluster`，则执行以下命令：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup cluster deploy prod-cluster v3.0.12 /tmp/topology.yaml
```

Comment:
@lucklove Is the version number correct? I think it should be `v4.0.0-rc` rather than `v3.0.12`.